controller/resource.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright   Copyright (C) 2011 - 2015 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://github.com/joomlatools/joomlatools-framework-activities for the canonical source repository
+ */
+
+/**
+ * Resource Controller.
+ *
+ * @author  Arunas Mazeika <https://github.com/amazeika>
+ * @package Koowa\Component\Activities
+ */
+class ComActivitiesControllerResource extends KControllerModel
+{
+    protected function _initialize(KObjectConfig $config)
+    {
+        if ($this->getIdentifier()->getPackage() != 'activities')
```

Comment:
In here the check makes sense. On LOGman I'm extending this controller. This check will see if I have a resources model on LOGman and if I don't it will create an alias so that we fallback to the Activities resources model. Same with permission.